apiFunctions.php

```diff
@@ -492,7 +492,26 @@ function expand_by_jstor($template) {
      $jstor = $match[1]; // remove ?seq= stuff
   }
   if (substr($jstor, 0, 1) === 'i') return FALSE ; // We do not want i12342 kind
-  $dat = @file_get_contents('https://www.jstor.org/citation/ris/' . $jstor);
+  
+  if (getenv('TRAVIS')) {
+    $jstor_opts = array(
+    'http'=>array(
+      'method'=>"GET",
+      'header'=>"Accept-language: en\r\n" .
+                "Cookie: UUID=3d1209ba-d7d6-40dc-8f1a-9190c982e0f2\r\n"
+    )
+    );
+  } else {
+   $jstor_opts = array(
+    'http'=>array(
+      'method'=>"GET",
+      'header'=>"Accept-language: en\r\n" .
+                "Cookie: UUID=2c114fa7-67a8-4291-a967-e4c5c5a563ac\r\n"
```

Comment:
This cookie will only exist on your local machine.  CURL has cookie-handling facilities.  But the appropriate thing here is to use JSTOR's API; if this route is not designed for programmatic access, then we should respect JSTOR's wishes and not overload their servers.

Reply:
I have an email into JSTOR.  @ms609 we can commit as-is for now which JSTOR on the toolforge turned off.